examples/react/src/registerServiceWorker.js

```diff
@@ -0,0 +1,117 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
Let's delete this file, it's not needed for understanding

Reply:
done.